pkg/validations/validations.go

```diff
@@ -16,8 +16,8 @@ import (
 )
 
 const (
-	baseDomainRegex          = "^([a-z0-9]+(-[a-z0-9]+)*)+$"
-	dnsNameRegex             = "^([a-z0-9]+(-[a-z0-9]+)*[.])+[a-z]{2,}$"
+	baseDomainRegex          = `^[a-z\d][\-]*[a-z\d]+$`
```

Comment:
@ammont82 please align it in the UI code

Reply:
@gamli75 https://github.com/openshift-assisted/assisted-installer-ui/pull/2268